examples/py/umaa_globalvectorcmd.py

```diff
@@ -87,4 +92,7 @@ def publisher_main():
 
     args = parser.parse_args()
 
-    publisher_main()
+    try:
+      publisher_main()
+    except KeyboardInterrupt:
+      pass
```

Comment:
Probably we can print a message similar to other C++ applications that we use "Finishing <task name>..."